src/app/system/system.service.ts

```diff
@@ -99,4 +99,51 @@ export class SystemService {
     return this.http.put(`/configurations/${configurationId}`, configuration);
   }
 
+  /**
+   * @returns {Observable<any>} Fetches Account Number Preferences.
```

Comment:
```suggestion
   * @returns {Observable<any>} Account number preferences.
```
Make similar change in all cases.